connection.js

```diff
@@ -267,6 +267,18 @@ class Connection {
             return prev ? prev[curr] : undefined
         }, this)
     }
+    set relaying (val) {
+        if (this.transaction) {
+            this.transaction._relaying = val;
+        }
+        else {
+            this._relaying = val;
+        }
+    }
+    get relaying () {
+        if (this.transaction && this.transaction._relaying) return this.transaction._relaying;
```

Comment:
What if you explicitly set relaying to false in a transaction hook, when it was true in the connection?

Maybe just do `if ('_relaying' in this.transaction)` instead?

Reply:
Then you've just set relaying=false in **only** the transaction. Which is basically what you want, isn't it?

If the connection was granted relaying permissions **outside** of a transaction, it should remain set to that for the duration of the connection. If it's modified for a particular transaction (a rare occurrence), shouldn't that change end with the transaction?

Reply:
Perhaps the conditional there should be !== undefined.

Reply:
Yeah that's exactly the same as using `in` fwiw.

Reply:
Oooooohhh, right. That's nicer syntax, I'll update.